app-shell/src/usb.ts

```diff
@@ -156,6 +157,13 @@ export function registerUsb(dispatch: Dispatch): (action: Action) => unknown {
           destroyUsbHttpAgent()
           ipcMain.removeHandler('usb:request')
           dispatch(usbRequestsStop())
+          // handle any additional invocations of usb:request
+          ipcMain.handle('usb:request', () =>
+            Promise.resolve({
+              status: 400,
+              statusText: 'Bad Request',
```

Comment:
```suggestion
              statusText: 'USB robot disconnected',
```

just to make somebody's future wtf a little more easily answered

Reply:
makes sense to me